### PR TITLE
NOISSUE - Add API key documentation

### PR DESCRIPTION
**Pull request by fbugarski**
Signed-off-by: Filip Bugarski <filipbugarski@gmail.com>

### What does this do?
Add API key documentation.
### Which issue(s) does this PR fix/relate to?
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -894,39 +894,63 @@ Date: Wed, 03 Nov 2021 13:00:05 GMT
...
-Content-Length: 273
+Content-Length: 476
 Connection: keep-alive
 Access-Control-Expose-Headers: Location
 
-{"value":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2Mzg0MDgyMzAsImlhdCI6MTYzODM3MjIzMCwiaXNzIjoibWFpbmZsdXguYXV0aCIsInR5cGUiOjB9.niDXlGC8G_NhyaT9WxlW_7srfsexOQd0bWTViJAObJU","issued_at":"2021-12-01T15:23:50.964773058Z","expires_at":"2021-12-02T01:23:50.964773058Z"}
+{"id":"4d62fb1e-085e-435c-a0c5-5255febfa35b","value":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2NDAwMzU1ODQsImp0aSI6IjRkNjJmYjFlLTA4NWUtNDM1Yy1hMGM1LTUyNTVmZWJmYTM1YiIsImlhdCI6MTYzOTkzNTU4NCwiaXNzIjoibWFpbmZsdXguYXV0aCIsInN1YiI6ImZscDFAZW1haWwuY29tIiwiaXNzdWVyX2lkIjoiYzkzY2FmYjMtYjNhNy00ZTdmLWE0NzAtMTVjMTRkOGVkMWUwIiwidHlwZSI6Mn0.RnvjhygEPPWFDEUKtfk5okzVhZzOcO0azr8gd5vby5M","issued_at":"2021-12-19T17:39:44.175088349Z","expires_at":"2021-12-20T21:26:24.175088349Z"}
 ```
 
 ### Gets API key details for the given key
```
Comment (dborovcanin) [inline docs/api.md]: Get API key details

**Review by dborovcanin — DISMISSED** [review]

**Review by blokovi — DISMISSED** [review]

File: docs/api.md [hunk 2]
```diff
@@ -893,40 +893,64 @@ Date: Wed, 03 Nov 2021 13:00:05 GMT
...
+Content-Length: 218
+Connection: keep-alive
+Access-Control-Expose-Headers: Location
+
+{"id":"f630f594-d967-4c54-85ef-af58efe8e8ed","issuer_id":"c93cafb3-b3a7-4e7f-a470-15c14d8ed1e0","subject":"test@email.com","type":2,"issued_at":"2021-12-19T17:42:40.884521Z","expires_at":"2021-12-20T21:29:20.884521Z"}
 ```
 
 ### Revoke API key identified by the given ID
 
 > Must-have: 'user_token' and 'key_id"
```
Comment (drasko) [inline docs/api.md]: You probably meant `'key_id'`. not `'key_id"`?

Comment (drasko) [reply docs/api.md]: Same above.

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/api.md (hunk 2 above)
Comment (fbugarski) [reply docs/api.md]: Yup :)

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**